### PR TITLE
feat(collector): apply job analyze API

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- apply job analyze API
image

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue [hunk 1]
```diff
@@ -192,15 +192,14 @@ const handleClickListItem = (detailLink) => {
...
 /* Watcher */
-// watch(() => state.items, async (value) => {
-//     const ids = value?.map((item) => item.collectorId);
-//     if (ids.length > 0) {
-//         const promises = ids.map(collectorPageStore.getCollectorSchedule);
-//         await Promise.all(promises);
-//     }
-// });
+watch(() => state.items, async (value) => {
+    const ids = value?.map((item) => item.collectorId);
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue]: nitpick: maybe need default value
```typescript
const ids = value?.map((item) => item.collectorId) ?? [];
```

File: apps/web/src/services/asset-inventory/collector/collector-main/collector-page-store.ts [hunk 2]
```diff
@@ -59,6 +59,39 @@ export const useCollectorPageStore = defineStore('collector-page', {
...
             }
         },
+        async setCollectorJobs(ids) {
+            try {
+                await SpaceConnector.clientV2.inventory.job.analyze({
+                    query: {
+                        filter: [
+                            {
+                                k: 'created_at',
+                                v: 'now - 5d',
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-main/collector-page-store.ts:59]: 😮

File: apps/web/src/services/asset-inventory/collector/model.ts [hunk 3]
```diff
@@ -133,3 +133,13 @@ export interface CollectorUpdatePluginParameter {
 export interface CollectorDeleteParameter {
     collectors: [string, ...string[]];
 }
+
+// JobAnalyzeModel
+export interface JobStatus {
+    job_id: string;
+    status: string;
+}
+export interface jobAnalyzeModel {
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/model.ts:142]: suggestion: Please use Pascal case for interface name.

File: apps/web/src/services/asset-inventory/collector/collector-main/collector-page-store.ts [hunk 4]
```diff
@@ -48,12 +49,35 @@ export const useCollectorPageStore = defineStore('collector-page', {
...
+                        fields: {
+                            job_status: {
+                                operator: 'push',
+                                fields: {
+                                    status: 'status',
+                                    job_id: 'job_id',
+                                },
+                            },
+                        },
+                    },
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-main/collector-page-store.ts:78]: praise: understandable!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue [hunk 5]
```diff
@@ -144,6 +144,18 @@ const state = reactive({
                     ]).rawQueryStrings,
                 },
             };
+
+            const matchedJobIndex = collectorPageState.collectorJobStatus.findIndex((status) => status.collector_id === d.collector_id);
+            const recentJobAnalyze = collectorPageState.collectorJobStatus[matchedJobIndex]?.job_status.slice(-5) || [];
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue:151]: suggestion: You can shorten this code using 'find' method. :)

```suggestion
            const matchedJob = collectorPageState.collectorJobStatus.find((status) => status.collector_id === d.collector_id);
            const recentJobAnalyze = matchedJob?.job_status.slice(-5) || [];
```

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue [hunk 6]
```diff
@@ -144,6 +144,18 @@ const state = reactive({
...
+
+            while (recentJobAnalyze.length < 5) {
+                const noneValue = {
+                    job_id: '',
+                    status: JOB_STATE.NONE,
+                };
+                recentJobAnalyze.unshift(noneValue);
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue:158]: question: Type Inference occurs in this case?

**Review by sulmoJ — APPROVED** [review]
LGTM!!

**Review by WANZARGEN — APPROVED** [review]
I left some question and suggestions!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue (hunk 6 above)
Comment (skdud4659) [reply apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue:158]: Although the type is specified in `CollectorItemInfo.vue`, which receives `state.item` as `props`, I will also specify it here!

**Pull request merged**